queries.sql

```diff
@@ -98,4 +98,31 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* Query multiple tables */
+SELECT name FROM animals JOIN owners ON owner_id =owner_id
+WHERE fullname = 'Melody Pond';
+
+SELECT * FROM animals JOIN species s ON species_id = s.id
+WHERE s.name = 'Pokemon';
+
+SELECT fullname AS owner, name AS pet FROM animals a
+FULL JOIN owners o ON o.id = a.owner_id;
+
+SELECT COUNT(a.name), s.name FROM animals a JOIN species s ON species_id = s.id
+GROUP BY s.name;
+
+SELECT a.name FROM animals a 
+LEFT JOIN owners o ON o.id = a.owner_id
+LEFT JOIN species s ON s.id = a.species_id
+WHERE o.fullname = 'Jennifer' AND s.name = 'Digimon';
+
+SELECT * FROM animals a
+FULL JOIN owners o ON o.id = owner_id
+WHERE o.fullname = 'Dean Winchester' AND escape_attempts = 0;
```

Comment:
- [ ] Good job adding the changes mentioned by the previous reviewer. However, I noticed that you didn't add add screenshots of their queries results to the pull request description. It is impossible to check if their results are correct. This will help us to ensure that you run the queries with success.